src/FunctionParametersHelper.hx

```diff
@@ -0,0 +1,254 @@
+package;
+import api.Completion.CompletionType;
+import api.Completion.CompletionResult;
+import js.codemirror.CodeMirror;
+import js.codemirror.CodeMirror.Pos;
+import api.Completion.CompletionItem;
+import js.Browser;
+import js.html.DivElement;
+import js.html.SpanElement;
+
+/**
+ * ...
+ * @author AS3
+ */
+class FunctionParametersHelper
+{
+	public var widgets:Array<LineWidget> = [];
+	var lastPos:Pos;
+	
+	static var instance:FunctionParametersHelper = null;
+	
+	public function new()
+	{
+			
+	}
+	
+	public static function get()
```

Comment:
same comment as for `Completion` : 
how about using the constructor, and keeping the instance in `Editor` ?